app/assets/src/components/views/compare/compound.js

```diff
@@ -0,0 +1,65 @@
+export default function compound() {
```

Comment:
Could you add a comment here and maybe name the function better?  Hard to understand what this function does.

Reply:
Moving it to `utils/scales`. The name makes sense in the context of scale.